src/stylesheets/components/_accordions.scss

```diff
@@ -1,12 +1,12 @@
 // Variables
 
-$accordion-border: 3px solid $color-theme-base-lighter;
+$accordion-border: units(2px) solid color('base-lighter');
```

Comment:
In this instance I think it might look better rounding up to 0.5 units (4px)

Reply:
Done in 64b23da. The latest Sketch file has it at 1px, so we'll need to update there as well.